doc/API.md

```diff
@@ -3,10 +3,13 @@
 revaultd exposes a [JSON-RPC 2.0](https://www.jsonrpc.org/specification)
 interface over a Unix Domain socket.
 
-| Command                     | Description                        |
-| --------------------------- | ---------------------------------- |
-| [`getinfo`](#getinfo)       | Display general information        |
-| [`listvaults`](#listvaults) | Display a paginated list of vaults |
+| Command                     | Description                               |
+| --------------------------- | ----------------------------------------- |
+| [`getinfo`](#getinfo)       | Display general information               |
+| [`getvault`](#getvault)     | Display specific information of a vault   |
+| [`listvaults`](#listvaults) | Display a paginated list of vaults        |
+| [`sigvault`](#sigvault)     | Sign the Revault pre-signed transactions  |
```

Comment:
Can we call it `signvault` ? I like to use `sig` to describe the actual signatures instead of the action of signing.